CHANGELOG.md

```diff
@@ -4,6 +4,80 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.12.0 - 2019-02-05
+
+### Changed
+
+- We now wait for [`ClusterVersion`][ClusterVersion] to report all
+  operators as available before returning from `create cluster`.
+- We now configure the network operator via
+  `networks.config.openshift.io` and reserve
+  `networkconfigs.networkoperator.openshift.io` for lower-level
+  configuration (although we still generate it as well).
+- We now set `apiServerURL` and `etcdDiscoveryDomain` in
+  `infrastructures.config.openshift.io`.
+- Release binaries are now stripped, which dramatically reduces their
+  size.  Builds with `MODE=dev` remain unstripped if you want to
+  attach a debugger.
+- On AWS, `destroy cluster` no longer depends directly on the cluster
+  name (although it still depends on the cluster name indirectly via
+  the `kubernetes.io/cluster/{name}` tag).  This makes it easier to
+  reconstruct `metadata.json` for `destroy cluster` if you
+  accidentally removed the file before destroying your cluster.
+- On AWS, the default worker MachineSets have been bumped to 120 GiB
+  volumes to increase our baseline performance from on [gp2's sliding
+  IOPS scale][aws-ebs-gp2-iops].  The new default worker volumes match
+  our master bump from 0.5.0.
+- On OpenStack, the HAProxy configuration on the service VM is
+  dynamically updated as masters and workers are added and removed.
+  This supports console access, among other things.
+- Several doc and internal cleanups.
+
+### Fixed
+
+- We no longer write distracting `ERROR: logging before flag.Parse...`
+  messages from our underlying Kubernetes libraries.
+- On loading `install-config.yaml`, we now error on CIDRs whose IP is
+  not at the beginning of the masked subnet.  For example, we now
+  error for 192.168.126.10/24, since the beginning of that subnet is
```

Comment:
WDYT about ticks around the cidrs and ips?

Reply:
> WDYT about ticks around the cidrs and ips?

Done (and for a historical entry too) with 6a97a58 -> 173930e.